--- opencv.js	2022-11-27 19:11:01.404462900 +0900
+++ opencv_kai.js	2022-12-01 17:42:46.208077500 +0900
@@ -20,7 +20,7 @@
     root.cv = factory();
   }
 }(this, function () {
-
+
 var cv = (function() {
   var _scriptDir = typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined;
   return (
@@ -40,9 +40,8 @@
       define([], function() { return cv; });
     else if (typeof exports === 'object')
       exports["cv"] = cv;
-
+
   if (typeof Module === 'undefined')
-    Module = {};
+    var Module = {};
   return cv(Module);
 }));
-
\ No newline at end of file
